_docs/00-getting-started.md

```diff
@@ -47,3 +75,13 @@ sh run.sh
 ## Try Infer in your browser
 
 Try Infer on a small example on [Codeboard](https://codeboard.io/projects/11587?view=2.1-21.0-22.0).
+
+# Next Sections
+
+[//]: # (These links are required because they are completely hidden and inaccessible when the web page is viewed on a tablet or phone)
```

Comment:
Ah, sorry about that, I'll see if we can fix this. We have the links you added here in the meantime.